<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <widget class="QCustomPlot" name="graph" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="min_x">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <double>-10000000000000000303786028427003666890752.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000044885712678075916785549312.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="max_x">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <double>-10000000000000000303786028427003666890752.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000044885712678075916785549312.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_min_x">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>min X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_max_x">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>max X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="max_y">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <double>-10000000000000000303786028427003666890752.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000044885712678075916785549312.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_max_y">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>max Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_min_y">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>min Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="min_y">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <double>-10000000000000000303786028427003666890752.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000044885712678075916785549312.000000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>MathCalc/View/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
